pkg/controllers/report/resource/controller.go

```diff
@@ -320,7 +320,8 @@ func (c *controller) updateDynamicWatchers(ctx context.Context) error {
 		for _, policy := range vapPolicies {
 			kinds, err := admissionpolicy.GetKinds(policy.Spec.MatchConstraints, restMapper)
 			if err != nil {
-				return err
+				logger.Error(err, "failed to resolve kinds for ValidatingAdmissionPolicy", policy.Name)
+				continue
```

Comment:
Why use `continue` here and in the other cases below?
Don't we want to process the valid kinds that were detected in the policy (eg. with `addGVKToGVRMapping`)?